modules/windows-node-services.adoc

```diff
@@ -22,6 +22,12 @@ The following Windows-specific services are installed on each Windows node:
 |Windows Machine Config Bootstrapper (WMCB)
 |Configures the kubelet and CNI plug-ins.
 
+|Windows Explorer
+|Provides a graphical user interface for accessing the file systems. 
```

Comment:
This is `windows_exporter which is a Prometheus exporter for Windows nodes.`

Reply:
https://github.com/openshift/prometheus-community-windows_exporter

Reply:
Sorry! I can't read apparently!